### PR TITLE
Additional Developer Section Fixes

### PR DESCRIPTION
**Pull request by fabioberger**
## Description

- Fix padding issue in `Developers` dropdown on Firefox
- Switch homepage image requested by Ben
- Move sidebar padding to scrollable content from container
- Use `media` module instead of writing it by hand
- Fix scroll lag on doc ref and wiki pages
- Remove unused `hashchange` listeners

## Checklist:

*   [ ] Prefix PR title with `[WIP]` if necessary.
*   [ ] Prefix PR title with bracketed package name(s) corresponding to the changed package(s). For example: `[sol-cov] Fixed bug`.
*   [ ] Add tests to cover changes as needed.
*   [ ] Update documentation as needed.
*   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/website/ts/components/nested_sidebar_menu.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            const finalLink = {
+                ...link,
+                title: menuItemTitle,
+            };
+            return <MenuItem key={`menu-item-${menuItemTitle}`} link={finalLink} />;
+        });
+        // tslint:disable-next-line:no-unused-variable
+        return (
+            <div key={`section-${sectionName}`} className="py1" style={{ color: colors.greyTheme }}>
+                <div style={{ fontSize: 14, letterSpacing: 0.5 }} className="py1 pl1">
```
Comment (fragosti) [inline packages/website/ts/components/nested_sidebar_menu.tsx]: `Text` ?

File: packages/website/ts/components/nested_sidebar_menu.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        const isActive = this.state.isActive;
+        return (
+            <Link
+                to={this.props.link.to}
+                shouldOpenInNewTab={this.props.link.shouldOpenInNewTab}
+                onActivityChanged={this._onActivityChanged.bind(this)}
+            >
+                <Button
+                    borderRadius="4px"
+                    padding="0.4em 6px"
```
Comment (fragosti) [inline packages/website/ts/components/nested_sidebar_menu.tsx]: The vertical padding scales with font but the horizontal padding is always 6px?

File: packages/website/ts/pages/wiki/wiki.tsx [hunk 3]
```diff
@@ -198,8 +194,4 @@ export class Wiki extends React.Component<WikiProps, WikiState> {
         }
         return sectionNameToLinks;
     }
-    private _onHashChanged(_event: any): void {
-        const hash = window.location.hash.slice(1);
-        sharedUtils.scrollToHash(hash, sharedConstants.SCROLL_CONTAINER_ID);
```
Comment (fragosti) [inline packages/website/ts/pages/wiki/wiki.tsx]: yassssss

File: packages/website/ts/components/nested_sidebar_menu.tsx (hunk 2 above)
Comment (fabioberger) [reply packages/website/ts/components/nested_sidebar_menu.tsx]: good catch, made both `em`

**Pull request merged**